Sources/MonkeyKing/MonkeyKing+Message.swift

```diff
@@ -0,0 +1,653 @@
+
+import Foundation
```

Comment:
It will be better if you can help to comment the changes when you separated the code to different file.

Reply:
I think it helps to clarify each actions.